src/main/java/com/sap/oss/phosphor/fosstars/model/score/oss/BanditScore.java

```diff
@@ -0,0 +1,83 @@
+package com.sap.oss.phosphor.fosstars.model.score.oss;
+
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.LANGUAGES;
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.RUNS_BANDIT_SCANS;
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.USES_BANDIT_SCAN_CHECKS;
+import static com.sap.oss.phosphor.fosstars.model.other.Utils.findValue;
+import static com.sap.oss.phosphor.fosstars.model.value.Language.PYTHON;
+
+import com.sap.oss.phosphor.fosstars.model.Value;
+import com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures;
+import com.sap.oss.phosphor.fosstars.model.score.FeatureBasedScore;
+import com.sap.oss.phosphor.fosstars.model.value.Languages;
+import com.sap.oss.phosphor.fosstars.model.value.ScoreValue;
+
+/**
+ * <p>The score shows if and how a project uses static analysis with Bandit.
+ * The score is based on the following features.</p>
+ * <ul>
+ *  <li>{@link OssFeatures#USES_BANDIT_SCAN_CHECKS}</li>
+ *  <li>{@link OssFeatures#RUNS_BANDIT_SCANS}</li>
+ *  <li>{@link OssFeatures#LANGUAGES}</li>
+ * </ul>
+ */
+public class BanditScore extends FeatureBasedScore {
+
+  /**
+   * Programming languages supported by Bandit.
+   *
+   * @see <a href="https://bandit.readthedocs.io/en/latest/#welcome-to-bandit-s-developer-documentation">Bandit
+   * overview</a>
+   */
+  private static final Languages SUPPORTED_LANGUAGES = Languages.of(PYTHON);
+
+  /**
+   * Defines how the score value is increased if a project runs Bandit scans.
+   */
+  private static final double BANDIT_SCANS_POINTS = 6.0;
+
+  /**
+   * Defines how the score value is increased if a project runs Bandit checks for commits.
+   */
+  private static final double BANDIT_CHECKS_POINTS = 7.0;
+
+  /**
+   * Initializes a new {@link BanditScore}.
+   */
+  BanditScore() {
+    super("How a project uses Bandit", USES_BANDIT_SCAN_CHECKS, RUNS_BANDIT_SCANS, LANGUAGES);
+  }
+
+  @Override
+  public ScoreValue calculate(Value<?>... values) {
+    Value<Boolean> usesBanditChecks = findValue(values, USES_BANDIT_SCAN_CHECKS,
+        "Hey! You have to tell me if the project uses Bandit checks!");
+    Value<Boolean> runsBanditScans = findValue(values, RUNS_BANDIT_SCANS,
+        "Hey! You have to tell me if the project runs Bandit scans!");
+    Value<Languages> languages = findValue(values, LANGUAGES,
+        "Hey! You have to tell me which languages the project uses!");
+
+    ScoreValue scoreValue = scoreValue(MIN,
+        usesBanditChecks, runsBanditScans, languages);
+
+    if (allUnknown(usesBanditChecks, runsBanditScans, languages)) {
+      return scoreValue.makeUnknown().explain(
+          "The score value is unknown because all required features are unknown.");
+    }
+
+    if (!languages.isUnknown() && !SUPPORTED_LANGUAGES.containsAnyOf(languages.get())) {
```

Comment:
If the languages feature is unknown, but the rest two features are false, then the scoring function produces 0 which may not be always correct. In this case, you may consider returning N/A but I am not sure though.

Reply:
It could also be that 
- The languages data provider fails to collect the language information but the project still triggers the Bandit scan. 
- I guess it is up to us to decide how we want to take this up.

Reply:
I have added a check for `unknown` language. I decided that it doesn't make sense to check for a Bandit scan if there is no mention that the project uses Python scripts.